common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -37,6 +38,38 @@ public void validateCreateUserRequest(Request userRequest) {
     validateWebPages(userRequest);
   }
 
+  private void validateVerificationSourceAndCode(Request userRequest) {
+    String verificationCode = (String) userRequest.getRequest().get(JsonKey.VERIFICATION_CODE);
+    String verificationSource = (String) userRequest.getRequest().get(JsonKey.VERIFICATION_SOURCE);
+    if ((StringUtils.isBlank(verificationCode) && StringUtils.isBlank(verificationSource))
```

Comment:
additional brackets not required - first && conditions will be evaluated - then ||